ming_sim/cli_backend.py

```
@@ -3997,6 +4008,11 @@ def _extract_secret_order(
         result["covert_task"] = covert_task
     if contract_error:
         result["contract_error"] = contract_error
+    # extract_failed 标记供下游分流：
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)